fe/fe-core/src/main/java/com/starrocks/connector/iceberg/rest/IcebergRESTCatalog.java

```diff
@@ -111,6 +111,9 @@ public IcebergRESTCatalog(String name, Configuration conf, Map<String, String> p
         OAuth2SecurityConfig securityConfig = OAuth2SecurityConfigBuilder.build(restCatalogProperties);
         OAuth2SecurityProperties securityProperties = new OAuth2SecurityProperties(securityConfig);
 
+        // Remove existing OAuth2 properties to avoid duplication before adding the processed ones
+        restCatalogProperties.entrySet().removeIf(entry -> 
```

Comment:
These OAuth2 properties already convert to iceberg REST catalog properties in  OAuth2SecurityProperties, they do not have same key, OAuth2 property is "oauth2.scope", after convert in OAuth2SecurityProperties, it's "scope".
Do you mean "oauth2.scope" in restCatalogProperties will result in polaris authentication failed?